models/taskRequests.js

```diff
@@ -76,13 +77,13 @@ const addOrUpdate = async (taskId, userId) => {
     if (taskRequestData) {
       const currentRequestors = taskRequestData.requestors;
       if (currentRequestors.length >= 1) {
```

Comment:
Take the condition on line `79` out. It will help in improving readability and from 2-levels of if nesting.